### DISCUSSION TITLE
Can OnlyOffice achieve features like this?

### DISCUSSION THREAD

**Discussion by langziyang** [ONLYOFFICE/DocumentServer#2902] [Q&A]
I want to insert my own component into Excel or Word, just like in the image. What I mean is using a plugin, as I am a plugin developer.
![image](https://github.com/user-attachments/assets/64060309-a234-4b0d-bb2d-d8bea60ca964)

**Comment by VladimirProkofiev52** [comment]
Hello! @langziyang 
I reviewed your request but wasn't able to fully understand what you're asking for. 
Could you please clarify what the object in question is? 
For example, is it an image or a diagram?  
To better assist you, we would need more information. 
If you have an example, please feel free to attach it.  

You might also find the answer to your question in our [documentation](https://api.onlyoffice.com/plugin/basic).

**Reply by langziyang** [reply]
hi,this is not image, i think it`s html component. there is the link:

[https://docs.qq.com/sheet/DUGFUemFScUpDemVS?tab=BB08J2](url)
